src/diagrams/class/classDb.js

```diff
@@ -345,8 +345,34 @@ const setDirection = (dir) => {
   direction = dir;
 };
 
+let title = '';
```

Comment:
Do we want to add the default "Class chart" title text here in case a user doesn't pass in anything?

Reply:
I am on the fence, if the title starts getting shown that is not ideal... Unless it gets the user to add a better title. I have been back and forth on this.

Reply:
I do think there should be some work to get titles to show up on every chart. Perhaps this is a carrot to making things more accessible?

Reply:
Ohhhh right - I forgot that this would be shown to all users 😬 Let's skip for now, but maybe we can revisit the option to have a title displayed for each chart. That would definitely help with accessibility!